web/app/js/components/TopModule.jsx

```diff
@@ -19,6 +19,8 @@ import _throttle from 'lodash/throttle';
 import _values from 'lodash/values';
 import { withContext } from './util/AppContext.jsx';
 
+const WS_NORMAL_CLOSURE = 1000;
+const WS_ABNORMAL_CLOSURE = 1006;
```

Comment:
My only styling nit - I'd add a line underneath line 23, since we typically add a new line under blocks of `const` declarations for readability (see `ErrorModal.jsx` for an example).

Reply:
Sure  :+1: done !